MUXSDKKaltura/MUXSDKKalturaTests/MockedData.swift

```diff
@@ -62,6 +62,40 @@ enum MockedData {
         )
     }
     
+    static var customerData3: MUXSDKCustomerData? {
```

Comment:
Okay I may be missing something here, I did not realize we were going down this pattern of adding a new numbered customer data object for each new test? Let's discuss in slack